lib/src/rules/use_setters_to_change_properties.dart

```diff
@@ -61,7 +61,8 @@ class _Visitor extends SimpleAstVisitor<void> {
         node.returnType?.type?.isVoid != true) {
       return;
     }
-    void _visitExpression(Expression expression) {
+
+    void visitExpression(Expression expression) {
```

Comment:
Does this now override `visitExpression` in `SimpleAstVisitor`?  Maybe another name to avoid confusion?

Reply:
A different name would be good because it obviously did create confusion and probably will again, but `SimpleAstVisitor` doesn't implement `visitExpression`. The method `visitExpression` is in `GeneralizingAstVisitor`.